infra/azure.tf

```diff
@@ -146,9 +146,13 @@ CRON
 chmod +x /root/daily-reset.sh
 
 cat <<CRONTAB >> /etc/crontab
-30 5 * * 1-5 root /root/daily-reset.sh '{"du1":10,"du2":0,"dw1":5,"dw2":0}' >> /root/log 2>&1
-30 18 * * 1-5 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
-30 5 * * 6,7 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
+30 5 * 1-11 1-5 root /root/daily-reset.sh '{"du1":10,"du2":0,"dw1":5,"dw2":0}' >> /root/log 2>&1
+30 18 * 1-11 1-5 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
+30 5 * 1-11 6,7 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
+30 5 1-22 12 1-5 root /root/daily-reset.sh '{"du1":10,"du2":0,"dw1":5,"dw2":0}' >> /root/log 2>&1
+30 18 1-22 12 1-5 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
+30 5 1-22 12 6,7 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
```

Comment:
This is the setup we currently have (10/5 machines during working hours on weekdays, 2/1 on weekends), but restricted to (lines 149-151) the first 11 months of the year, then (lines 152-153) the first 22 days of the twelfth month.

The next line sets the weekend configuration (2/1) for all the remaining days (23 through 31 of month 12).